docs/multicluster/user-guide.md

```diff
@@ -464,18 +481,39 @@ For example, once you export the `default/nginx` Service in member cluster
 `default/antrea-mc-nginx` will be created in `test-cluster-east`, as well as
 a ServcieImport CR with name `default/nginx`. Now, Pods in `test-cluster-east`
 can access the imported Service using its ClusterIP, and the requests will be
-routed to the backend `nginx` Pods in `test-cluster-west`.
+routed to the backend `nginx` Pods in `test-cluster-west`. You can check the
+imported Service and ServiceImport with commands:
+
+```bash
+$kubectl get serviceimport antrea-mc-nginx -n default
+NAME            TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
+antrea-mc-nginx ClusterIP   10.107.57.62 <none>        443/TCP   10s
+
+$kubectl get serviceimport nginx -n default -o yaml
+apiVersion: multicluster.x-k8s.io/v1alpha1
+kind: ServiceImport
+metadata:
+  name: nginx
```

Comment:
I removed annotations. I do not know `multicluster.antrea.io/local-cluster-id` should be west or east (I thought east, but wrote west). And it is not important for users anyway. We should update the CRD yaml to let kubectl print unnecessary information for the CRDs with the table format.

BTW, what `multicluster.antrea.io/local-cluster-id` is used for? I did not get why we need to annotate the local cluster ID to ServiceImport.

Reply:
Here is the table samples:
```bash
$kubectl get gateway -A
NAMESPACE     NAME          AGE
kube-system   lan-k8s-1-1   3d2h

$kubectl get serviceimport -A
NAMESPACE     NAME      TYPE           IP                AGE
default       nginx     ClusterSetIP   ["10.19.57.62"]   43d
kube-system   nginx     ClusterSetIP   ["10.19.27.13"]   47d

$kubectl get clusterinfoimport -A
NAMESPACE     NAME                            AGE
kube-system   test-cluster-east-clusterinfo   9d
```
You can remove the annotation, I will to double check what's this annotation for.

Reply:
Ok. I changed SeviceImport to table format.